spec/lib/rmt/cli/import_spec.rb

```diff
@@ -117,14 +124,20 @@
 
 
     context 'with existing lockfile' do
-      before { allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError) }
+      before do
+        allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError)
```

Comment:
Same as with the other spec: If possible, create a real lockfile and test with that!

Reply:
changed to stubbing the `flock` method